### PR TITLE
Add a setting to disable notifications

### LINKED ISSUE #281: Add a setting for disabling native notifications

### PR DESCRIPTION
**Pull request by dkniffin**
Fixes #281

### CODE REVIEW

File: src/js/actions/AppActions.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ const init = () => {
         utils.getMetadata(path).then((track) => {
             ipcRenderer.send('playerAction', 'trackStart', track);
 
-            if(!app.browserWindows.main.isFocused()) {
+            if(!app.browserWindows.main.isFocused() && app.config.get('displayNotifications')) {
```
Comment (KeitIG) [inline src/js/actions/AppActions.js]: This check should be in `NotificationsActions`, not here

Comment (dkniffin) [inline src/js/actions/AppActions.js]: Done.

File: src/js/actions/SettingsActions.js [hunk 2]
```diff
@@ -127,6 +127,15 @@ const refreshProgress = (percentage) => {
     });
 };
 
+const toggleDisplayNotifications = (value) => {
+    app.config.set('displayNotifications', value);
+    app.config.saveSync();
```
Comment (YurySolovyov) [inline src/js/actions/SettingsActions.js]: We should stop doing sync IO in most UI-related things someday

Comment (YurySolovyov) [inline src/js/actions/SettingsActions.js]: Not a blocker for this PR though

Comment (KeitIG) [inline src/js/actions/SettingsActions.js]: There's an async method too `.save(callback)`.

It's fine for now as we don't do massive blocking stuff.

**Pull request merged**